airflow/providers/apache/spark/hooks/spark_submit.py

```diff
@@ -111,6 +111,7 @@ def __init__(
         executor_memory: str | None = None,
         driver_memory: str | None = None,
         keytab: str | None = None,
+        use_krb5ccache: bool = False,
```

Comment:
Should we make it a keyword-only argument? Also, could we add it to the docstring?

```python
    *,
    use_krb5ccache: bool = False,
```

Reply:
Sure